bundles/org.openhab.binding.ipcamera/src/main/java/org/openhab/binding/ipcamera/internal/handler/IpCameraHandler.java

```diff
@@ -525,6 +525,18 @@ private void checkCameraConnection() {
             }
             return;// ffmpeg snapshot stream is still alive
         }
+
+        // if ONVIF cam also use connection state which is updated by regular messages to camera
+        if (thing.getThingTypeUID().getId().equals(ONVIF_THING) && snapshotUri.isEmpty()) {
+            logger.trace("is ONVIF with empty snapshot URI");
+            if (onvifCamera.isConnected()) {
+                logger.trace("cam is connected");
```

Comment:
Same comment as above, we do not need a log entry every time 'nothing is wrong'. There is a cost to performance if everything in your smart home started doing this, even if the logging level is not TRACE, there is a cost to check what the log level is.